bool use_defaults
float32 x_del # Delivery location coordinates
float32 y_del
float32 z_del
bool delete_prev_traj
---
bool success
string message
